phys2bids/physio_obj.py

```diff
@@ -298,8 +298,15 @@ def __getitem__(self, idx):
 
             idx = slice(idx, idx + 1)
 
+        # If idx.start or stop are None, make them 0 or trigger length.
+        if not idx.start:
+            idx.start = 0
+        if not idx.stop:
+            idx.stop = trigger_length
```

Comment:
Slicer attributes are non-modifiable, so you're going to want to handle this a bit differently. I'd suggest something like:

```python
if idx.start is None:
    idx = slice(0, idx.stop)
if idx.stop is None:
    idx = slice(idx.start, trigger_length)
```

It would also be good to update the doc-string since currently this function cannot accept a tuple-of-ints.